packages/@ama-sdk/create/src/index.it.spec.ts

```diff
@@ -1,32 +1,32 @@
+/**
```

Comment:
Should not be `\*` instead?

Reply:
I don't think so
https://jestjs.io/docs/configuration#testenvironment-string

Reply:
Does it work with simple comment then? Because a meanful comment to an `import` statement it is a bit strange

Reply:
No, I've changed the code to associate the comment with a `const` representing the model exported by the custom Jest Environment